src/main/java/com/mateacademy/lessons13/rest/MateGroupServiceImpl.java

```diff
@@ -80,4 +82,77 @@ public Response removeStudent(@PathParam("groupId")int groupId, @PathParam("surn
 
 		return Response.status(ACCEPTED).entity(mateGroup).type(MediaType.APPLICATION_JSON).build();
 	}
+
+	@Override
+	@PUT
+	@Path("/humanResources")
+	@Consumes(MediaType.APPLICATION_JSON)
+	@Produces(MediaType.APPLICATION_JSON)
+	public Response addHR(@PathParam("groupId")int groupId, HumanResource newHR) {
+
+		MateGroup mateGroup = mateGroups.get(groupId);
+		if (mateGroup != null) {
+			mateGroup.getHumanResources().add(newHR);
+			return Response.status(ACCEPTED).entity(mateGroup).type(MediaType.APPLICATION_JSON).build();
+		}
+		return Response.status(Status.NOT_FOUND).build();
+	}
+
+	@Override
+	@DELETE
+	@Path("/hrs/{surname}")
```

Comment:
You use different pathes in methods. In previous POST you used "/humanResources", in this method you use just "/hrs".
It's critical.

Reply:
I dont agree with you that it is critical, I try humanResources or /hrs -my programm  work

Reply:
It's critical. You work with resources. Hard understanding your's clients why you did that